tests/lib/snaps/test-snapd-physical-memory-control/physical-mem.c

```diff
@@ -0,0 +1,58 @@
+#include <fcntl.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#define MAP_SIZE 4096UL
+#define MAP_MASK (MAP_SIZE - 1)
+
+int main(int argc, char** argv)
+{
+    int fd;
+    void *map_base, *virt_addr;
+    uint32_t read_result, write_result;
+
+    uint32_t writeval = 't';
+    off_t address = 0x00000001;
```

Comment:
I'm not sure that writing to this address is a good idea. Best consult memory map for x86 in this case.

BTW. will this be allowed STRICT_DEVMEM?

Reply:
I second this opinion. What was the expected outcome? How do we expect people to use this interface?